micronaut-camunda-bpm-feature/src/test/kotlin/info/novatec/micronaut/camunda/bpm/feature/test/AdminUserCreatorTest.kt

```diff
@@ -101,31 +103,74 @@ class AdminUserCreatorTest {
             assertAdminGroupExists(processEngine)
             assertAdminGroupAuthorizationsExist(processEngine)
         }
+    }
 
-        fun queryUser(processEngine: ProcessEngine, userId: String): User {
-            return processEngine.identityService.createUserQuery().userId(userId).singleResult()
-        }
+    @MicronautTest
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    inner class AdminUserCreatorTestWithOnlyRequiredProperties : TestPropertyProvider {
```

Comment:
please also rename "AdminUserCreatorTestWithProperties" to "AdminUserCreatorTestWithAllProperties"